docs: update setup and calibration (#3)
tier4/drs-docs: docs/calibration/index.md

        6. When the number of detected pairs is over the predefined value, the “Save calibration” button will become available. After collecting sufficient data, press the button and save the result into a yaml file. After confirming that the result is correctly saved, close all windows. 
            ![](images/image-20241121-122343.png)

> 5. Copy the result file to the corresponding ECU with the proper renaming.

Reviewer: ```suggestion
5. Copy the resulting file to the corresponding ECU with the proper renaming.
```

    ```
7. After calling the above service, the tool starts alignment (this may take a while). Once the alignment finishes, the “Save calibration” button on the third dialog will become available. If the button is enabled, press it and save the result.
    ![](images/image-20241127-142231.png)

8. Copy the result file to the **both** ECUs with the proper renaming.

>    - Because the result will be referred in both ECUs, keep in mind not to miss copying it to the both ECUs.

Reviewer: ```suggestion
   - Because the file will be used in ECU0 and ECU1, please make sure to copy to both.
```

   - Rename the result file to `drs_base_link_to_lidars.yaml`
   - The replacement target looks like:

   ```bash
   data_recording_system/src/individual_params/config/default
   ├── drs_base_link_to_lidars.yaml # <- replace the contents of this file
   └...
   ```

# Design values between `base_link` and `drs_base_link` 
- In DRS, `base_link` is described as the center between the vehicle's two rear wheels, while `drs_base_link` stands for the coordinate system origin of the INS module on the roof.

>   - Both has the same coordinate criteria; `x` faces forward of the vehicle, `y` faces left side of the vehicle, and `z` faces the sky

Reviewer: ```suggestion
  - Both have the same coordinate criteria; `x` faces forward of the vehicle, `y` faces left side of the vehicle, and `z` faces the sky.
```

> - There is no need for a precise pose relationship between them, but rough values will be appreciated to enhance the value of collected data by the vehicle. To meet the requirment, fill the pose of the `drs_base_link` relative to the `base_link` by designed values, which are taken using something like CAD. The target file is:

Reviewer: ```suggestion
- There is no need for a precise pose relationship between them, but rough values will be appreciated to enhance the value of collected data by the vehicle. To meet this requirement, fill the pose of the `drs_base_link` relative to the `base_link` with the design values, which are calculated using CAD. The target file is:
```
